dts/bindings/i2s/intel,ssp-ports.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: Intel SSP ports available
+
+compatible: "intel,ssp-ports"
+
+include: base.yaml
+
+properties:
+    base:
+      type: int
+      description: Number of base SSP ports available.
+
+    extended:
+      type: int
+      description: Number of extended SSP ports available.
```

Comment:
Wouldn't be better having ssp-clk-gating and ssp-ports merged ? I think that is preety similar to `adsp-settings`.

Reply:
I'm not entirely sure `adsp-settings` is ideal, being a catch all for "global settings". Aren't these specific binds clearer?

Reply:
Yeah, these ssp properties all need to be part of one compatible.